qiita_db/test/test_meta_util.py

```diff
@@ -46,13 +46,13 @@ def test_get_accessible_filepath_ids(self):
         obs = qdb.meta_util.get_accessible_filepath_ids(
             qdb.user.User('shared@foo.bar'))
         self.assertItemsEqual(obs, {
-            1, 2, 3, 4, 5, 9, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21})
+            1, 2, 3, 4, 5, 9, 12, 16, 17, 18, 19, 20, 21})
```

Comment:
Why are these being deleted? Are the analysis permissions be different from now on?